src/dct/systems/loadouts.lua

```diff
@@ -10,8 +10,20 @@ local enum     = require("dct.enum")
 local dctutils = require("dct.utils")
 local settings = _G.dct.settings
 
--- returns totals for all weapon types, returns nil if the group
--- does not exist
+local isAAMissile = {
+	[Weapon.MissileCategory.AAM] = true,
+	[Weapon.MissileCategory.SAM] = true,
```

Comment:
Why is SAM in the category of air-to-air missile?

Reply:
Defensive coding against DB re-use, say if the Ka-50 gets Iglas or the AH-64 gets Stingers that are classified internally as SAMs. Doubtful, but a SAM strapped onto an aircraft doesn't fit the bill of an air-to-ground weapon.

I can remove the condition and either collapse it into a single equality check or leave the table with only one item if you feel it's too speculative.

Reply:
Hummm.... is there any way for this to be abused with standard non-modded games? In other words would this allow someone to take a weapon that they would otherwise not be able too?

Reply:
Only through cheating or editing missions files directly. Anything swapped in the miz will be loaded in the plane, and some modules can fire anything they're carrying, even if they have no avionics integration with it.